practice-app/server/src/routes/users/users.route.js

```diff
@@ -19,7 +19,7 @@ const usersRouter = express.Router();
  *     parameters:
  *      - in: "body"
  *        name: "Body"
- *        description: "descc"
+ *        description: "Creates a user with given data."
  *        required: true
  *        schema:
  *          type: object
```

Comment:
Your endpoints do not work on swagger. Please remove parameters and change it to requestBody as I explained.

Reply:
Swagger definitions updated, They work now.